docs/src/documentation/05-development/03-hooks/uselockscrolling.mdx

```diff
@@ -1,22 +1,24 @@
 ---
 title: useLockScrolling
-description: Prevent scrolling the page and only allow it for the given element, usually an overlay.
+description: Prevent scrolling the page and only allow it for the given element, usually an overlay
 ---
 
 import UseLockScrollingReadMe from "@kiwicom/orbit-components/src/hooks/useLockScrolling/README.md";
 import { Alert } from "@kiwicom/orbit-components";
 import { MDXProvider } from "@mdx-js/react";
 
+export const Warning = ({ children }) => (
```

Comment:
It can be exported, I do not mind that. I did it because of the new eslint rule that will not allow arrow functions inside Provider and it requires it to be an separate component, but in mdx it won't it show correctly without export.